op-node/cmd/genesis/cmd.go

```diff
@@ -141,6 +141,14 @@ var Subcommands = cli.Commands{
 		Action: func(ctx *cli.Context) error {
 			deployConfig := ctx.Path("deploy-config")
 			log.Info("Deploy config", "path", deployConfig)
+			// if DEPLOY_CONFIG_SNAP is specified, we want to check that deployConfig is not changed.
+			snapPath := os.Getenv("DEPLOY_CONFIG_SNAP")
```

Comment:
This is an  example of code that is not maintainable. This randomly reads an env var where all other env vars come thru the cli config